bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -44,173 +45,123 @@ __version__ = pycbc.version.git_verbose_msg
-    inj_data['time'] = get_column(inj_table, 'geocent_end_time') \
-                + 1e-9 * get_column(inj_table, 'geocent_end_time_ns')
+        missed_data[param] = inj_data['/missed/%s' % param][...]
+        found_data[param] = inj_data['/found/%s' % param][...]
+
+    # Calculate injections mchirp
+    missed_data['mchirp'] = mchirp_from_mass1_mass2(missed_data['mass1'],
+                                                    missed_data['mass2'])
+    found_data['mchirp'] = mchirp_from_mass1_mass2(found_data['mass1'],
+                                                   found_data['mass2'])
+
+    # Injections time
+    found_data['time'] = inj_data['/found/tc'][...]
+    missed_data['time'] = inj_data['/missed/tc'][...]
+
+    # Calculate effective distance for the ifos
+    found_data['eff_dist'] = 0
+    missed_data['eff_dist'] = 0
+    for ifo in ifos:
+        antenna = Detector(ifo)
+        found_data['eff_dist_%s' % ifo] = antenna.effective_distance(
+                                    found_data['distance'],
+                                    found_data['ra'],
+                                    found_data['dec'],
+                                    found_data['polarization'],
+                                    found_data['time'],
+                                    found_data['inclination']
+                                    )
+        missed_data['eff_dist_%s' % ifo] = antenna.effective_distance(
+                                    missed_data['distance'],
+                                    missed_data['ra'],
+                                    missed_data['dec'],
+                                    missed_data['polarization'],
+                                    missed_data['time'],
+                                    missed_data['inclination']
+                                    )
+        found_data['eff_dist'] += 1.0 / found_data['eff_dist_%s' % ifo]
+        missed_data['eff_dist'] += 1.0 / missed_data['eff_dist_%s' % ifo]
+    found_data['eff_dist'] = 1.0 / found_data['eff_dist']
+    missed_data['eff_dist'] = 1.0 / missed_data['eff_dist']
+
+    # Get recovered parameters and statistic values for the found injections
+    # Recovered parameters
+    rec_params = ['mass1', 'mass2', 'spin1z', 'spin2z']
+    for param in rec_params:
+        found_data['rec_%s' % param] = bank_file[param][inj_data['network/template_id']]
+
+    found_data['end_time_gc'] = \
+        inj_data['/network/end_time_gc'][...]  # needed to use bestNR
+    found_data['rec_mchirp'] = mchirp_from_mass1_mass2(
+        found_data['rec_mass1'],
+        found_data['rec_mass2'])
+    # Staistics value
+    filter_stats = ['coherent_snr', 'reweighted_snr', 'null_snr']
+    for param in filter_stats:
+        found_data[param] = inj_data['/network/%s' % param][...]
+    found_data['chisq'] = inj_data['/network/my_network_chisq'][...]
+    found_data['nifos'] = inj_data['/network/nifo'][...].astype(int)
```

Comment:
From `inj_data['/network/']` you also need to get 'ra' and 'dec'

Reply:
Not so sure that 'ra' and 'dec' are found in the 'network' group